src/frontend/src/instance/distributed.rs

```diff
@@ -547,6 +515,30 @@ impl DistInstance {
         .context(error::RequestMetaSnafu)
     }
 
+    async fn drop_table_procedure(
+        &self,
+        table_name: &TableName,
+        table_id: TableId,
+    ) -> Result<SubmitDdlTaskResponse> {
+        let request = SubmitDdlTaskRequest {
+            task: DdlTask::new_drop_table(
+                table_name.catalog_name.to_string(),
+                table_name.schema_name.to_string(),
+                table_name.table_name.to_string(),
+                table_id,
+            ),
+        };
+
+        timeout(
+            // TODO(weny): makes timeout configurable.
```

Comment:
Lots of `timeout` paramaters are can't be configured right now in distribute mode, i think we should pay this tech debet ASAP. @MichaelScofield

Reply:
We can provide a separate channel_manager for ddl-client and configure timeout independently.

Reply:
tracked in #1935